src/providers/FileSystemProvider/FileSystemProvider.ts

```diff
@@ -554,7 +557,7 @@ export class FileSystemProvider implements vscode.FileSystemProvider {
     }
     // Write the new file
     // This is going to attempt the write regardless of the user's response to the check out prompt
-    const api = new AtelierAPI(oldUri);
+    const api = new AtelierAPI(newUri);
```

Comment:
What bug is this fixing? We prevent renaming across workspace folders so the connection info should be the same between `oldUri` and `newUri`.

Reply:
When Testing Manager copies a .cls file from a file:// uri to an isfs:// one.